public/translations/en-US.json

```diff
@@ -883,11 +895,72 @@
     "success": "Settings saved successfully",
     "community_info": {
       "title": "Community Information",
-      "description": "Logo, name, description"
+      "description": "Logo, name, description, url",
+      "guidance": "Guidance",
+      "logo": {
+        "title": "Community logo",
+        "description": "Your file must be in PNG, JPG or SVG format. It should measure 80px in width and 80px in height to be better viewed on the interface."
+      },
+      "invitation": {
+        "title": "Access to the community",
+        "description": "When a community \"invitation\" is set, new members only get access through an invitation."
+      },
+      "cover_photo": {
+        "title": "Community cover picture",
+        "description": "Be sure to add a picture that has good quality"
+      },
+      "fields": {
+        "name": "Community name",
+        "description": "What is the community",
```

Comment:
maybe "what this community is about" or "Describe this community"

Reply:
what gives a weird meaning, in portuguese it would be something like "o que é a comunidade?", idk know sounds bad